trello_service.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import configparser
+import board as b
+import logging
+
+config = configparser.ConfigParser()
+config.read('.cfg')
+trello = config['TRELLO']
```

Comment:
This isn't documented in the README.md

Also, is there any reason you are using ConfigParser instead of environment variables?

Reply:
No special reason to use ConfigParser. Since there were several options available I just picked one up